docs/apache-airflow/tutorial.rst

```diff
@@ -376,3 +376,133 @@ Here's a few things you might want to do next:
       - Review the :ref:`List of operators <pythonapi:operators>`
       - Review the :ref:`Macros reference<macros>`
     - Write your first pipeline!
+
+
+Lets look at another example; we need to get some data from a file which is hosted online and need to insert into our local database. We also need to look at removing duplicate rows while inserting.
+
+Initial setup
+'''''''''''''
+We need to have docker and postgres installed.
+We will be using this `docker file <https://airflow.apache.org/docs/apache-airflow/stable/start/docker.html#docker-compose-yaml>`_
+Follow the instructions properly to set up Airflow.
+
+Create a Employee table in postgres using this
+
+.. code-block:: bash
```

Comment:
Does `sql` work better?

Reply:
The query is run on the psql CLI, so should I update it to SQL or let it be as bash only?

Reply:
Aren't commands run *in* psql SQL (dialet)? Like how commands run in the `python` REPL are Python code, not Bash.